tests/test_adaption_prompt.py

```diff
@@ -78,7 +106,19 @@ def test_attributes(self) -> None:
         self.assertTrue(hasattr(model, "from_pretrained"))
         self.assertTrue(hasattr(model, "push_to_hub"))
 
+        #Test Mistral
+        if self.mistral_available:
```

Comment:
Instead of attaching the mistral tests to the llama tests, could you please create a separate test for each? You can decorate them with `@unittest.skipIf(not is_mistral_available())` to avoid the `if self.mistral_available:` line.

Reply:
Thank you for the advice! I will do it. Good feedback is very helpful for me to learn and apply the best practices :)